pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -715,10 +716,22 @@ func takeByTopologyNUMADistributed(topo *topology.CPUTopology, availableCPUs cpu
 				})
 			}
 
+			// If alignBySocket is enabled, combo in same socket will be considered firstly.
 			// If the best "balance score" for this combo is less than the
 			// lowest "balance score" of all previous combos, then update this
 			// combo (and remainder set) to be the best one found so far.
-			if bestLocalBalance < bestBalance {
+			if alignBySocket {
```

Comment:
would it be possible to reformat the code to avoid nested ifs and use `return Continue` like the original code is? If possible it should be a bit more readable/consistent

Reply:
Done.